prims_algorithm.cpp

```diff
@@ -0,0 +1,48 @@
+#include <bits/stdc++.h>
+using namespace std;
```

Comment:
Add algorithm name, sample description, input output of tha algorithm in comments

Reply:
not here comments I meant in the code..comment the input output and the idea of algorithm in the code not here